src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__content">
+    <div class="product-card" data-qa="card">
+      <div class="product-card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac picture"
+          class="product-card__item"
+        >
+      </div>
+
+      <div class="product-card__text">
+        Моноблок APPLE A1419 iMac
+        27" Retina 5K (MNED2UA/A)
+      </div>
+
+      <div class="product-card__code">Код товара: 195434</div>
```

Comment:
it's better to use semantic tags

Reply:
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 11th item